pkg/utils/util.go

```diff
@@ -250,3 +250,11 @@ func CheckBackplaneVersion(cmd *cobra.Command) {
 
 	logger.WithField("Current version", info.Version).WithField("Latest version", latestVersion).Warn("Your Backplane CLI is not up to date. Please run the command 'ocm backplane upgrade' to upgrade to the latest version")
 }
+
+func PrettyPrint(i interface{}) string {
```

Comment:
This function is implemented but never referenced?  ( I like this function :D )

Reply:
Oops, that was missed in clean up, it's a nifty function tho...

Removing.